src/bin.js

```diff
@@ -41,21 +40,17 @@ function parseOptions(args, availableOptions) {
       console.log("We suggest using the new 'overrides' option instead of nested .babelrc's, can check out http://new.babeljs.io/docs/en/next/babelrc.html#overrides");
       console.log("");
     }
-    paths.forEach(p => writeBabelRC(p, upgradeOptions));
+
+    for (let path of paths)  writeBabelRC(path, upgradeOptions);
   }
 
-  mochaOpts.forEach(p => writeMochaOpts(p, upgradeOptions));
+  for (let path of mochaOpts)  writeMochaOpts(path, upgradeOptions);
 
   // TODO: allow passing a specific path
   await writePackageJSON(upgradeOptions);
 
-  // TODO: add smarter CLI option handling if we support more options
-  if (cliOptions.installDeps) {
-    if (cliOptions.write) {
-      console.log('Installing new dependencies');
-      await installDeps();
-    } else {
-      console.log('Run babel-upgrade with --write (or) -w and --install for it to install the newly added dependencies');
```

Comment:
Is this warning covered by `Commander`? As in, can Commander itself say something like `--install has to be provided with --write`?